docs/advanced-configuration.md

```diff
@@ -780,3 +780,35 @@ agent:
   * Cluster receiver is a 1-replica deployment of Open-temlemetry collector.
   * As any available node can be selected by the Kubernetes control plane to run the cluster receiver pod (unless we explicitly specify the `clusterReceiver.nodeSelector` to pin the pod to a specific node), `hostPath` or `local` volume mounts wouldn't work for such envrionments.
   * Data Persistence is currently not applicable to the k8s cluster metrics and k8s events.
+
+### Using OpenTelemetry eBPF helm chart with Splunk OpenTelemetry Collector for Kubernetes
```

Comment:
Question: Is there a set of generic instructions that customers will be able to leverage if they are using the upstream helm chart instead?

Reply:
discussed offline